src/commands/images/removeImage.ts

```diff
@@ -21,9 +21,9 @@ export async function removeImage(context: IActionContext, node?: ImageTreeItem,
 
     let confirmRemove: string;
     if (nodes.length === 1) {
-        confirmRemove = localize('vscode-docker.commands.images.remove.confirmSingle', 'Are you sure you want to remove image "{0}"? This will remove all matching and child images.', nodes[0].label);
+        confirmRemove = localize('vscode-docker.commands.images.remove.confirmSingle', 'Are you sure you want to remove image "{0}"? This will remove all matching images.', nodes[0].fullTag);
```

Comment:
This only shows when there is one image selected, not more than one (`if (nodes.length === 1)`). Before, if you asked to delete `mcr.microsoft.com/dotnet/core/sdk:3.1`, it would ask you if you wanted to delete `3.1`. That isn't helpful at all, since tons of images in the .NET Core family end with 3.1.

Reply:
Make sense.